ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsSet.scala

```diff
@@ -190,18 +190,19 @@ object MetricsSet {
   def countTreeTransactionsEvents(response: GetTransactionTreesResponse): Long =
     response.transactions.foldLeft(0L)((acc, tx) => acc + tx.eventsById.size)
 
-  private def optionalMetrics[T](configO: Option[CommonObjectivesConfig]): List[Metric[T]] =
-    configO.flatMap(createTotalRuntimeMetricO[T]).toList
-
-  private def createTotalRuntimeMetricO[T](config: CommonObjectivesConfig): Option[Metric[T]] = {
-    config.maxTotalStreamRuntimeDuration.map((maxStreamDuration: FiniteDuration) =>
-      TotalStreamRuntimeMetric.empty(
-        clock = Clock.systemUTC(),
-        startTime = Clock.systemUTC().instant(),
-        objective = MaxDurationObjective(maxValue = toJavaDuration(maxStreamDuration)),
-      )
+  private def optionalMetrics[T](configO: Option[CommonObjectivesConfig]): List[Metric[T]] = {
```

Comment:
Seem overly generic `maxTimeMetrics` ?

Reply:
Also why the change from Option to List, only seems to return an optional value ?

Reply:
Changing the name.
The `List` is convenient because we're appending the (possibly empty) result to another lists